packages/website/src/components/editor/LoadedEditor.tsx

```diff
@@ -159,7 +159,9 @@ export const LoadedEditor: React.FC<LoadedEditorProps> = ({
       onEsASTChange(webLinter.storedAST);
       onTsASTChange(webLinter.storedTsAST);
       onScopeChange(webLinter.storedScope);
-      onSelect(sandboxInstance.editor.getPosition());
+
+      const position = sandboxInstance.editor.getPosition();
+      onSelect(position ? tabs.code.getOffsetAt(position) : undefined);
```

Comment:
[Question] What's the difference between calling `onSelect` with the position vs. the offset position? I can't figure it out locally.

Reply:
this was done to simplify logic used to highlight elements `getPosition` returns class that contains lint/column and functions to mutate/validate those values

conversion between this structure and what we need previously was happening during comparison for every node, (this was actually more complex but lets leave at that), 

this changes that we are always sending `number` as cursor position within text and receiving [number, number] as position of selected node